.hadolint.yaml

```diff
@@ -0,0 +1,2 @@
+ignored:
+    - SC1091
```

Comment:
Why do we need this?

Reply:
I've added this for an old ShellCheck failure. Actually removed that problematic code, hence can remove this file completely.

Reply:
Done.

Reply:
👌